t/01-test_needle.t

```diff
@@ -413,6 +413,22 @@ subtest 'click point' => sub {
     is_deeply($needle, undef, 'multiple click points not accepted');
 };
 
+subtest 'workround property' => sub {
+    my $workaround_string_needle = needle->new($misc_needle_dir . 'check-workround-bsc1234567-20190522.json');
+    my $workaround_hash_needle   = needle->new($misc_needle_dir . 'check-workround-hash-bsc7654321-20190522.json');
+    my $no_workaround_needle     = needle->new($misc_needle_dir . 'click-point-center.json');
+
+    is($workaround_string_needle->has_property("workaround"), 1, "workaround property found when it is recorded in string");
```

Comment:
If you just want to test for a "true" value I suggest to use `ok` instead of `is`